### DISCUSSION TITLE
Fix unit representation

### DISCUSSION THREAD

**Discussion by fbordignon** [hgrecco/pint#1991] [Q&A]
Hello everyone, I would like to fix a unit representation the way I choose. For example Acoustic Impedance is often written as Density * Velocity (g/cm**3 * m/s) Keeping that order for the unit is nice because people look at it and see density at the first term (g/cm**3) and velocity (m/s) at the second. How can I fix that when printing the unit?
I always get  `gram * meter / centimeter ** 3 / second` which is the same but not quite intuitive. 

Thanks

**Comment by andrewgsavage** [comment]
thisis possible with a custom format in the master branch (from github, not released on pypi yet)
https://pint.readthedocs.io/en/latest/user/formatting.html#custom-formats

do post the code you make for acoustic impediance, it would be a good example for the docs.

**Reply by keewis** [reply]
the simpler version (using `register_unit_format`) has been in `pint` for quite some time now, so that wouldn't need the development version of `pint`.

**Reply by fbordignon** [reply]
Ok, let me pick your brains a little bit more. I see that the pint.formatter has a sorted option. For my case (acoustic impedance), I can register a custom unit format that: 1- order the unit items the way I like it 2- return a call to the formatter with sorted off.
Would that do the trick?


My project is at version 0.14 of pint that sorts the items automagically. It should be no problem updating to a newer version or even the dev version. Thank you very much!
